ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1452,6 +1452,28 @@ def verify_consumer_resources():
     log.info(f"Monitoring endpoint of cephcluster yaml: {monitoring_endpoint}")
     assert re.match("\\d+(\\.\\d+){3}", monitoring_endpoint)
 
+    ocs_version = version.get_semantic_ocs_version_from_config()
+
+    # Verify the default Storageclassclaims
```

Comment:
Please add this also to the docstring of verify_consumer_resources

Reply:
Added.